### ISSUE TITLE
Remove aliases to Pytorch implementations

### ISSUE THREAD

**Issue by gallego-posada** [cooper-org/cooper#19]
## Enhancement

Remove existing aliases to objects in `torch.optim`. Update documentation accordingly.

## Motivation

Having these aliases makes the interface opaque for the user: it is hard to tell that `cooper.optim.SGD` is in fact the same implementation as `torch.optim.SGD`.

**Comment by juan43ramirez** [comment]
I agree. This would result in Extragradient-capable optimizers coming from cooper.optim (e.g. cooper.optim.ExtraSGD) and "vanilla" optimizers coming from torch.optim (e.g. torch.optim.SGD). 
This is an inconvenience, but I believe the gain in clarity warrants the change.

**Comment by gallego-posada** [comment]
What is the inconvenience? The fact that the extra-optimizers need to be retrieved from cooper and not torch? 
This seems reasonable to me though -- torch never claims to have implementations of extra-gradient; so there would be no place for confusion on the user's side.

**Comment by juan43ramirez** [comment]
That is the inconvenience. I agree that removing the aliases is the way to go.
